starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
+<link rel="stylesheet" href="./css/main.css">
 <head>
   <title>Apple Pie Recipe</title>
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <header class="header">
+    <h1>Apple Pies</h1>
+  </header>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+<section id="intro">
+  <p>This was my grandmother's apple pie recipe. I have never seen another one
+  quite like it. It will always be my favorite and has won me several first place
+  prizes in local competitions. I hope it becomes one of your favorites as well!
+  </p>
+  <center><img class="img_recipeinfo" src="./images/recipe-info.png" alt="recipe-info pic"></center>
+  <hr >
+</section>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+<section id="Ingredients">
+  <h2>Ingredients</h2>
+  <ul>
+    <li>- 1 recipe pastry for a 9 inch double crust pie</li>
+    <li>- 1/2 cup unsalted butter</li>
```

Comment:
If you want to style your bullet points there's a more "official" way to do it, with the `list-style-type` property. It has quite a lot of values, you can learn more about it here: https://developer.mozilla.org/fr/docs/Web/CSS/list-style-type